.readthedocs.yaml

```diff
@@ -21,3 +27,5 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+  apt_packages:
+    - graphviz
```

Comment:
Is this something we need to add to ``requirements/docs.txt`` instead? https://pypi.org/project/graphviz/

Reply:
mail reply is hard to read so I will put this directly here: yes it needs to be here, python package doesn't install the underlying conversion utility which comes from apt package.